tests/processes/ProcessingTest.py

```diff
@@ -607,24 +608,24 @@ def test_global_instantiation(self):
         class TestOneBear(Bear):
 
             def __init__(self, file_dict, section, queue, timeout=0.1,
```

Comment:
use `*args` and `**kwargs`

And also these bears can be moved into `tests/test_bears/runtime_error_bears/`

Reply:
I will create an issue after this code gets merged, If no one takes the issue within a week or two I will solve that issue :+1: